### ISSUE TITLE
Quiz for OR Perceptron

### ISSUE THREAD

**Issue by abhijitapte** [udacity/dlnd-issue-reports#353]
The answer for quiz on OR perceptron is wrong.
Answer should be increase weights and increase bias.
To test this out, I used the example from And Perceptron which had weights (weight1 = 1.0 and weight2 = 0.2) and bias = -1.2

Since I needed to convert AND to an OR, I started altering the weights.

The combination that worked for me (of course you need to set the correct_outputs to [False, True, True, True])

combination 1: weight1=6.0, weight2=6.2, bias=-1.2
combination 2: weight1=1.0, weight2=0.2, bias=-0.2

In other words, in combination 1, you ended up increasing weights and in combination 2, you ended up keeping weights the same and increased bias slightly.

**Comment by abhijitapte** [comment]
<img width="888" alt="screen shot 2017-08-05 at 12 36 52 pm" src="https://user-images.githubusercontent.com/2939847/28994222-7f76c136-79e6-11e7-8ead-7cb821a54b85.png">
<img width="884" alt="screen shot 2017-08-05 at 12 37 22 pm" src="https://user-images.githubusercontent.com/2939847/28994220-7f7297e6-79e6-11e7-90a6-b994d02ee8db.png">
<img width="887" alt="screen shot 2017-08-05 at 12 38 51 pm" src="https://user-images.githubusercontent.com/2939847/28994221-7f76336a-79e6-11e7-8461-6173c1f95672.png">
<img width="891" alt="screen shot 2017-08-05 at 12 39 23 pm" src="https://user-images.githubusercontent.com/2939847/28994224-7f785794-79e6-11e7-9760-ea4c779cbe8f.png">
<img width="892" alt="screen shot 2017-08-05 at 12 40 07 pm" src="https://user-images.githubusercontent.com/2939847/28994225-7f7d7be8-79e6-11e7-8b40-5ff588129780.png">
<img width="889" alt="screen shot 2017-08-05 at 12 40 26 pm" src="https://user-images.githubusercontent.com/2939847/28994223-7f771a32-79e6-11e7-8a26-76eedd7dcac0.png">

**Comment by mcleonard** [comment]
Internal tracking: DLND-1160
